bin/conform-templates.php

```diff
@@ -0,0 +1,299 @@
+#!/usr/bin/env php
+<?php
+
+require __DIR__ . '/../vendor/autoload.php';
+
+if (! file_exists('composer.json')) {
+    fwrite(
+        STDERR,
+        'File composer.json does not exist, please call script from main repository directory'
+    );
+    exit(1);
+}
+
+$composer = json_decode(file_get_contents('composer.json'), true);
+if (! isset($composer['name']) || ! preg_match('/^([a-z-]+)\/([a-z-]+)$/', $composer['name'], $m)) {
+    fwrite(
+        STDERR,
+        'Cannot extract repository name from composer.json, please check value of "name" key in that file.'
+    );
+    exit(1);
+}
+$org = $m[1];
+$repo = $m[2];
+
+// if there is no docs directory create one
+if (! is_dir('docs')) {
+    // if there is doc directory rename it to docs
+    if (is_dir('doc')) {
+        rename('doc', 'docs');
```

Comment:
Hm... I think I wrote my comment here, but I can't see it now. I've tested it on couple repositories and it seems that GIT recognizes rename automatically. I can change it to system command if you prefer.

Reply:
I was thinking more in terms of the steps needed to run this script.

They become:

- run script
- remove old entries (`git rm -rf doc`)
- add new entries (`git add doc`)

While git will detect these correctly as renames, it's three operations, instead of one, which is my main concern.